src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -366,6 +392,22 @@ object SnowflakeTelemetry {
     }
     metric
   }
+
+  private[snowflake] def detectSparkLanguage(sparkConf: SparkConf): String = {
+    if (sparkConf.contains("spark.r.command")
+      || sparkConf.contains("spark.r.driver.command")
+      || sparkConf.contains("spark.r.shell.command")) {
+      "R"
+    } else if (sparkConf.contains("spark.pyspark.python")) {
+      sparkConf.get("spark.pyspark.python").split("/").last
```

Comment:
@sfc-gh-ema `sparkConf.get("spark.pyspark.python").split("/").last` will get the binary name. It may include the version number, but it is not reliable. What about we just use "Python"?